posix.mak

```diff
@@ -528,21 +528,21 @@ $(DSCANNER_DIR):
 	git -C $@ checkout $(DSCANNER_HASH)
 	git -C $@ submodule update --init --recursive
 
-$(DSCANNER_DIR)/dsc: | $(DSCANNER_DIR) $(DMD) $(LIB)
+$(DSCANNER_DIR)/dscanner: | $(DSCANNER_DIR) $(DMD) $(LIB)
 	# debug build is faster, but disable 'missing import' messages (missing core from druntime)
 	sed 's/dparse_verbose/StdLoggerDisableWarning/' $(DSCANNER_DIR)/makefile > $(DSCANNER_DIR)/dscanner_makefile_tmp
 	mv $(DSCANNER_DIR)/dscanner_makefile_tmp $(DSCANNER_DIR)/makefile
-	DC=$(abspath $(DMD)) DFLAGS="$(DFLAGS) -defaultlib=$(LIB)" $(MAKE) -C $(DSCANNER_DIR) githash debug
+	DC=$(abspath $(DMD)) DFLAGS="$(DFLAGS) -defaultlib=$(LIB)" $(MAKE) -C $(DSCANNER_DIR) githash release
```

Comment:
remove `release`. This is used for building the binaries uploaded to GH.

Reply:
If I remove "release", no dscanner binary is generated.

Reply:
it's in the bin/ subfolder

Reply:
Add "all" seems to work.